backend/cmd/cleanup/moneycollector/moneycollector.go

```diff
@@ -0,0 +1,77 @@
+package moneycollector
+
+import (
+	"kubecloud/internal"
+	"kubecloud/models"
+	"sync"
+
+	"github.com/rs/zerolog/log"
+	substrate "github.com/threefoldtech/tfchain/clients/tfchain-client-go"
+)
+
+type MoneyCollector struct {
+	db              models.DB
+	config          internal.Configuration
+	substrateClient *substrate.Substrate
+}
+
+const (
+	MinBalanceThreshold = 1e5
+)
+
+func NewMoneyCollector(db models.DB, config internal.Configuration, substrateClient *substrate.Substrate) *MoneyCollector {
+	return &MoneyCollector{
+		db:              db,
+		config:          config,
+		substrateClient: substrateClient,
+	}
+}
+
+func (m *MoneyCollector) CollectMoney() {
+	system, err := substrate.NewIdentityFromSr25519Phrase(m.config.SystemAccount.Mnemonic)
+	if err != nil {
+		log.Error().Err(err).Msg("MoneyCollector: failed to create system identity")
```

Comment:
Shouldn't it already be created?
Please update the error message. maybe `failed to load system identity`